docs/_posts/2018-06-26-v0-1-19-release.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: "v0.1.19 Dispatch Release"
+year: 2018
+---
+
+# v0.1.19 Dispatch Release
+
+[Download v0.1.19](https://github.com/vmware/dispatch/releases/tag/v0.1.19)
+
+
+## Support for Exposed Event Driver
+Events may now be pushed (as opposed to pulled) from event sources to Dispatch.  An *exposed* driver will create an ingress route and service to recieve events.
+When creating event driver type, use `--expose` to create exposed drivers, these drivers will have a URL which events can be pushed to.
+```bash
+$ dispatch create eventdrivertype ticker vmware/dispatch-ticker-driver:v0.1.0 --expose
```

Comment:
We should probably use a different example.  The ticker doesn't make sense to expose at it doesn't listen or receive events.

Just change it to reference the azure eventgrid driver:

https://github.com/dispatchframework/dispatch-events-eventgrid